docs/mkdocs.yml

```diff
@@ -87,3 +91,7 @@ extra:
     - icon: social-discussion
       link: https://github.com/protectai/nbdefense/discussions
   generator: false
+
+extra_javascript:
+  - assets/js/timeago.min.js
```

Comment:
Is there a way we could pull this from [a cdn](https://cdnjs.com/libraries/timeago.js/2.0.2)?

Reply:
Resolved comments!
Imported `lightgallery` from cdn as well.

- https://cdnjs.cloudflare.com/ajax/libs/timeago.js/4.0.2/timeago.min.js
- https://cdnjs.cloudflare.com/ajax/libs/lightgallery-js/1.4.1-beta.0/js/lightgallery.min.js